src/views/credits/credits.jsx

```diff
@@ -71,7 +71,12 @@ var Credits = React.createClass({
                         <img src="//cdn.scratch.mit.edu/get_image/user/2286560_170x170.png" alt="Carmelo Avatar" />
                         <span className="name">Carmelo Presicce</span>
                     </li>
-
+                    
+                    <li>
+                         <img src="//cdn2.scratch.mit.edu/get_image/user/default_170x170.png" alt="Tina Avatar" />
```

Comment:
Thanks! Tina has set an avatar. Can you update this to reflect that?

Reply:
Might need the URL unless there's an easy way to find that.